src/less/page-notice/page-notice.less

```diff
@@ -112,9 +112,13 @@ span[role="region"].page-notice {
     text-align: right;
 }
 
-/* support legacy 6.5 notice with heading + paragaphs */
 .page-notice__main p {
     font-size: @font-size-regular;
+    margin: 2px 0 0;
+}
+
+/* support legacy 6.5 notice with heading + paragaphs */
+.page-notice__main .page-notice__title + p {
```

Comment:
Just curious:

So when we have a paragraph in a title, we apply different spacing, at least that's what this selector is doing.

Personally I think we probably should never have a paragraph inside a title. Inside title we should only have a heading. (if this is what you were talking about earlier, then I agree, we shouldn't include a paragraph in a title. Only headings. I might have been confused...).

The question I have: do we ever have a paragraph in a title, or any examples with that? If not, I think we can probably remove this part and have all paragraphs remain the same.

Reply:
Got it
I think the plus only selects the first child. I think you have to use a ~ to select all children after title.

Reply:
Right. Forgot there could be multiple `<p>`. Fixed.